.changeset/serious-kids-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': patch
+---
+
+Fixes a bug in the Netlify Adapter where prerendered 404.astro pages weren't shown
```

Comment:
Would love for this to have a *tiny* bit more context, if it's a simple one liner...

> Fixes a bug in the Netlify Adapter where prerendered `404.astro` pages weren't displayed when/if...

If you can't answer that sentence in a few words, then it's fine! This feels like a bug that if people had it, they knew about it.  :smile: 

(If this didn't *always* happen, but the description of when it did is complicated or "it depends", you could say "where sometimes prerendered..." but not blocking.)

Reply:
@Skn0tt can you update the changeset?

Reply:
I do! I'm sorry, got lost in a bunch of other work.

Reply:
```suggestion
Fixes a bug in the Netlify Adapter where prerendered 404.astro pages weren't shown on hybrid/server deployments.
```